lib/tasks/taxonomy/analysis/taxons_tagged_to_content.rake

```diff
@@ -24,20 +24,33 @@ namespace :taxonomy do
         page += 1
       end
 
-      puts %w(organisations content_id base_path number_of_taxons).to_csv
+      content_items.each do |item|
+        details = []
+        item["taxons"].each do |taxon|
+          resp = Services.publishing_api.get_content(taxon)
+          details << resp["base_path"]
+        end
+        item["base_paths"] = details
+      end
 
-      content_items.each do |content_item|
-        organisations = content_item.fetch('organisations', [])
-          .map { |org| org['title'] }
-          .join('; ')
+      puts %w(Link Number_of_taxons Base_path_1 Taxon_id_1 Base_path_2
+        Taxon_id_2 Base_path_3 Taxon_id_3).to_csv
 
+      content_items.each do |content_item|
         csv_row = [
           content_item['link'],
           content_item['taxons'].length,
         ]
 
-        content_item['taxons'].each { |taxon_id| csv_row << taxon_id }
+        max = [content_item['base_paths'].length, content_item['taxons'].length].max
+
+        for i in 0...max
```

Comment:
This is completely fine, but not widely used in Ruby. One nice method on `Integer` is [upto](https://ruby-doc.org/core-2.2.0/Integer.html#method-i-upto). In this case, you could write is as:

```
0.upto(max) do |i|
  ...
end
```